hash_practice/exercises.py

```diff
@@ -27,3 +74,4 @@ def valid_sudoku(table):
     """
     pass
 
+grouped_anagrams(["eat", "tae", "tea", "eta", "aet", "ate"])
```

Comment:
Looks like there's an extra line here, if you don't need it you can get rid of it.